tests/integration/hive_metastore/test_pipeline_migrate.py

```diff
@@ -0,0 +1,156 @@
+    )
+
+    pipeline_names = [
+        f"pipeline-{make_random(4).lower()}-{watchdog_purge_suffix}",
+        f"pipeline-{make_random(4).lower()}-{watchdog_purge_suffix}",
+        f"skip-pipeline-{make_random(4).lower()}-{watchdog_purge_suffix}",
+    ]
+    created_pipelines = [
+        make_pipeline(
+            configuration=_PIPELINE_CONF,
+            name=pipeline_names[0],
+            target=runtime_ctx.make_schema(catalog_name="hive_metastore").name,
+            libraries=[PipelineLibrary(notebook=NotebookLibrary(path=dlt_notebook_paths[0]))],
+            clusters=[
+                PipelineCluster(
+                    node_type_id=ws.clusters.select_node_type(local_disk=False, min_memory_gb=16),
+                    label="default",
+                    num_workers=1,
+                    custom_tags={"cluster_type": "default", "RemoveAfter": watchdog_purge_suffix},
+                )
+            ],
+        ),
+        make_pipeline(
+            configuration=_PIPELINE_CONF,
+            name=pipeline_names[1],
+            target=runtime_ctx.make_schema(catalog_name="hive_metastore").name,
+            libraries=[PipelineLibrary(notebook=NotebookLibrary(path=dlt_notebook_paths[1]))],
+            clusters=[
+                PipelineCluster(
+                    node_type_id=ws.clusters.select_node_type(local_disk=False, min_memory_gb=16),
+                    label="default",
+                    num_workers=1,
+                    custom_tags={"cluster_type": "default", "RemoveAfter": watchdog_purge_suffix},
+                )
+            ],
+        ),
+        make_pipeline(
+            configuration=_PIPELINE_CONF,
+            name=f"skip-{pipeline_names[2]}",
+            target=runtime_ctx.make_schema(catalog_name="hive_metastore").name,
+            libraries=[PipelineLibrary(notebook=NotebookLibrary(path=dlt_notebook_paths[0]))],
+        ),
+    ]
+    ws.pipelines.start_update(created_pipelines[0].pipeline_id)
+    ws.pipelines.start_update(created_pipelines[1].pipeline_id)
+    assert (
+        ws.pipelines.wait_get_pipeline_idle(created_pipelines[0].pipeline_id, timeout=timedelta(minutes=6))
+    ).state == PipelineState.IDLE
+    assert (
+        ws.pipelines.wait_get_pipeline_idle(created_pipelines[1].pipeline_id, timeout=timedelta(minutes=2))
+    ).state == PipelineState.IDLE
+
+    job_with_pipeline_list = [
+        runtime_ctx.make_job(
+            tasks=[
+                Task(
+                    pipeline_task=PipelineTask(pipeline_id=created_pipelines[0].pipeline_id),
+                    task_key=make_random(4).lower(),
+                )
+            ]
+        ),
+        runtime_ctx.make_job(
+            tasks=[
+                Task(
+                    pipeline_task=PipelineTask(pipeline_id=created_pipelines[0].pipeline_id),
+                    task_key=make_random(4).lower(),
+                )
+            ]
+        ),
+    ]
+    pipelines = runtime_ctx.pipelines_crawler.snapshot()
```

Comment:
Scope the crawler to only crawl the given ids similar to the other `include_` parameters, then remove for loop below

Reply:
Added a TECH DEBT ticket for this: https://github.com/databrickslabs/ucx/issues/3399